test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/VerifyCommandTests.cs

```diff
@@ -19,9 +19,9 @@ namespace NuGet.CommandLine.FuncTest.Commands
     [Collection("Sign Command Test Collection")]
     public class VerifyCommandTests
     {
-        private const string _noTimestamperWarningCode = "NU3550";
-        private const string _primarySignatureInvalidErrorCode = "NU3030";
-        private const string _signingDefaultErrorCode = "NU3000";
+        private readonly string _noTimestamperWarningCode = NuGetLogCode.NU3027.ToString();
```

Comment:
What is the benefit of readonly over const here?

Reply:
The benefit is that `readonly` compiles while `const` does not.  :)

You can't write:

    private const string _noTimestamperWarningCode = NuGetLogCode.NU3027.ToString();

...because the initializer is not a compile-time constant expression.